### PR TITLE
Add Ha-Ri core team devs SSH key to all servers

### LINKED ISSUE #180: Make sure all sys admin SSH keys are shared on all servers
## Need

Instance manager want to be reassure that if issue and their sys admin is not available someone else from global pool can intervene.
For that we need all sys admin SSH on all instances production servers.

## Acceptance criteria

All global sys admin (Rob, Maikel, Matt, François, Pau, Enrico) have access to all OFN instance servers.

### PR DESCRIPTION
**Pull request by sauloperez**
Closes #180 

If there's a production issue in any of the OFN instances any Ha or Ri
dev can put the sysadmin hat on and fix it.

Access to production machines is also needed to investigate production
logs as well as production data when trying to fix high severity issues,
like happened recently.

### CODE REVIEW

File: inventory/group_vars/fr.yml [hunk 1]
```diff
@@ -24,3 +24,4 @@ users_sysadmin:
   - pau
   - enricostn
   - luis
+  - kristina
```
Comment (pacodelaluna) [inline inventory/group_vars/fr.yml]: 👍 Welcome @kristinalim

**Comment by sauloperez** [comment]
are we missing any instance's approval?

**Comment by sigmundpetersen** [comment]
US?

**Comment by sauloperez** [comment]
You got it; my favorite bot :joy:

File: inventory/group_vars/fr.yml (hunk 1 above)
Comment (kristinalim) [reply inventory/group_vars/fr.yml]: Thank you, @pacodelaluna! :slightly_smiling_face:

**Comment by sauloperez** [comment]
Waiting for US's ACK

**Comment by lauriewayne** [comment]
If I am authorized to ACK, please consider this change approved for the US instance. Thank you!

**Comment by sauloperez** [comment]
ok, now we need to reprovision all these servers. Please note down below which one you're going to be in charge of and check it once done.

- [ ] UK:
- [ ] US:
- [ ] AUS:
- [ ] DE:
- [ ] FR:
- [ ] Katuma: Pau

**Pull request merged**